src/entity/SourceMember.ts

```diff
@@ -18,7 +18,6 @@ export type SourceMemberFlagsPublic = Pick<
 >;
 
 @Entity()
-@Index('IDX_source_member_userId_flags_hideFeedPosts', { synchronize: false })
```

Comment:
This one might be invalid if it was not hit, I would expect it hit from `feedToFilters` so just keep a lookout on the tool if it reports anything for this flag. Maybe got broken after PG migration.